packages/client/src/index.ts

```diff
@@ -149,7 +153,7 @@ export default class LogtoClient {
   }
 
   private get tokenSetCacheKey() {
-    return `${TOKEN_SET_CACHE_KEY}::${this.oidcConfiguration.issuer}::${this.clientId}`;
+    return `${TOKEN_SET_CACHE_KEY}::${this.oidcConfiguration.issuer}::${this.clientId}::${this.scope}`;
```

Comment:
@IceHe create a issue replace spaces in scope as it is being used as party of the key.